meta-lxatac-bsp/recipes-bsp/barebox/files/patches/0011-scripts-add-bareboxtlv-host-target-tool.patch

```diff
@@ -225,8 +225,7 @@ index 000000000000..eca430104af9
 +			if (ret < 0)
 +				goto out;
 +		}
-+		if (ret >= 0)
-+			ret = write_full_verbose(1, p, size);
++		ret = write_full_verbose(1, p, size);
```

Comment:
This change is fine (because ret is never negative here), but I'd mention that in the commit log, at least with something like "simplification in bareboxtlv host tool".